ProjectLighthouse/Startup.cs

```diff
@@ -9,58 +13,129 @@
 using Microsoft.Extensions.Configuration;
 using Microsoft.Extensions.DependencyInjection;
 using Microsoft.Extensions.Hosting;
+using Microsoft.Extensions.Logging;
 
-namespace LBPUnion.ProjectLighthouse {
-    public class Startup {
-        public Startup(IConfiguration configuration) {
+namespace LBPUnion.ProjectLighthouse
+{
+    public class Startup
+    {
+        public Startup(IConfiguration configuration)
+        {
             this.Configuration = configuration;
         }
 
         public IConfiguration Configuration { get; }
 
         // This method gets called by the runtime. Use this method to add services to the container.
-        public void ConfigureServices(IServiceCollection services) {
+        public void ConfigureServices(IServiceCollection services)
+        {
             services.AddControllers();
             services.AddMvc(options =>
                 options.OutputFormatters.Add(new XmlOutputFormatter()));
-            
+
             services.AddDbContext<Database>();
         }
 
         // This method gets called by the runtime. Use this method to configure the HTTP request pipeline.
-        public void Configure(IApplicationBuilder app, IWebHostEnvironment env) {
-            if(env.IsDevelopment()) {
+        public void Configure(IApplicationBuilder app, IWebHostEnvironment env)
+        {
+            var computeDigests = true;
+            var serverDigestKey = Environment.GetEnvironmentVariable("SERVER_DIGEST_KEY");
+            if (string.IsNullOrWhiteSpace(serverDigestKey))
+            {
+                Logger.Log(
+                    "SERVER_DIGEST_KEY environment variable wasn't set, so server digest headers won't be set. This will break LBP 1 and LBP 3."
```

Comment:
```suggestion
                    "The SERVER_DIGEST_KEY environment variable wasn't set, so digest headers won't be set or verified. This will prevent LBP 1 and LBP 3 from working. To increase security, it is recommended that you find and set this variable."
```